pkg/devicemapper/devmapper_wrapper.go

```diff
@@ -206,6 +206,7 @@ func dmGetNextTargetFct(task *cdmTask, next unsafe.Pointer, start, length *uint6
 		*params = C.GoString(Cparams)
 	}()
 
+	//lint:ignore SA4000 false positive on (identical expressions on the left and right side of the '==' operator) (staticcheck)
 	nextp := C.dm_get_next_target((*C.struct_dm_task)(task), next, &Cstart, &Clength, &CtargetType, &Cparams)
```

Comment:
Spent 5 minutes looking at this and I could not imagine how the checker could come up seeing `==` in here :)

Reply:
Could it be that detection devicemapper package is problematic because of CGO? 

We can create a tracking issue of these false positives (I assume they want a minimal reproducer, which might take some time to construct?)